src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -339,7 +339,7 @@ private static string RemovePII(object arg)
             if (arg is Exception ex && ex.GetType().FullName.StartsWith("Microsoft.IdentityModel.", StringComparison.Ordinal))
                 return ex.ToString();
 
-            return arg.GetType().ToString();
+            return arg?.GetType().ToString();
```

Comment:
Can you also add a check in FormatInvariant above.
if args is null, then we will throw a null ref.

Reply:
@RojaEnnam ill make this change and push the commit.